.github/scripts/Dockerfile

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+FROM fedora:34 as builder
+
+# FROM quay.io/centos/centos:stream8 as builder
+# Enable additional package repositories for CentOS
+#  note: PowerTools is called CodeReady Linux Builder in RHEL 8
+# RUN dnf -y install epel-release && dnf config-manager --set-enabled powertools && dnf -y install gcc gcc-c++ make cmake cyrus-sasl-devel cyrus-sasl-plain cyrus-sasl-gssapi cyrus-sasl-md5 openssl-devel libuuid-devel nodejs swig wget patch findutils git valgrind libwebsockets-devel python3-devel libnghttp2-devel && dnf clean all -y
+RUN dnf -y install gcc gcc-c++ make cmake cyrus-sasl-devel cyrus-sasl-plain cyrus-sasl-gssapi cyrus-sasl-md5 openssl-devel libuuid-devel nodejs swig wget patch findutils git valgrind libwebsockets-devel python3-devel libnghttp2-devel && dnf clean all -y
```

Comment:
cyrus-sasl-plain cyrus-sasl-gssapi cyrus-sasl-md5? These are needed for running tests, should not be needed for build.

nodejs? Not needed, we don't have webconsole anymore.

valgrind? Not needed here.

Reply:
Those cyrus-sasl-* were useful in the former upstream image, but I believe we can get rid of them now.
Same for nodejs, since there is no more console here.

Reply:
The cyrus sasl plugins could be useful in the image, but I agree there is no need to have them in the builder.

Reply:
Actually at least cyrus-sasl-plain is required at present.

Reply:
Do you mean in the builder? The console has been removed, so do you think it should be kept in the final image?

Reply:
If the console is gone then you are right, the existing use case is gone. We can always add it when needed.